src/python/pants/backend/go/go_sources/BUILD

```diff
@@ -1,4 +1,5 @@
 # Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-resources(sources=["*.go", "!gentestflags.go"])
+python_source(name="init", source="__init__.py")
+resources(sources=["*.go", "!gentestflags.go"], dependencies=[":init"])
```

Comment:
Why not use `tailor` here?

Reply:
Tailor didn't suggest it because it was a "solitary init" file. I think that option was a bad idea...it's tripped up people multiple times and the sole motivation was fewer BUILD files.